docs/rules/arrow-linebreak.md

```diff
@@ -0,0 +1,84 @@
+# Enforce the location of arrow function bodies with implicit returns (arrow-linebreak)
+
+An arrow function body can contain an implicit return as a single statement instead of a block. It can be useful to enforce a consistent location for the arrow function body.
```

Comment:
Nitpick: The body of implicit-return arrow functions is actually an *expression*, not a statement.